README.md

```diff
@@ -1,11 +1,25 @@
-# Deadlands Classic
-
-A Foundry VTT Game System for playing Deadlands Classic, Deadlands Hell On Earth Classic, and (hopefully) Deadlands Lost Colony Classic.
-
-For Foundry Virtual Tabletop v11.304+.
-
-Document Version: 20230702
-
-<hr/>
-
-This FVTT Game System uses the revised Deadlands Classic rules from all three Deadlands Classic Games, as represented in the combined set of published Deadlands Classic rulebooks, with the later rules and rulebooks taking precedence over earlier ones.
\ No newline at end of file
+# Deadlands Classic
+
+A Foundry VTT Game System for playing Deadlands Classic, Deadlands Hell On Earth Classic, and (hopefully) Deadlands Lost Colony Classic.
+
+For Foundry Virtual Tabletop v11.306+.
+
+Document Version: 20230722
+
+<hr/>
+
+## Original Deadlands Rules 
+
+This FVTT Game System uses the revised Deadlands Classic rules from all three Deadlands Classic Games, as represented in the combined set of published Deadlands Classic rulebooks, with the later rules and rulebooks taking precedence over earlier ones.
+
+## Copyright &amp; License
+
+Copyright (c) 2023 {{Andrew}}, {{Andy}}, {{Ben}}, Matthew J Black, {{Haxxx}}  & {{James}}.

```

Comment:
This isn't normally a part of GPL, but sure 🙂 

```suggestion
Copyright (c) 2023 {{Andrew}}, Andrew Herron, {{Ben}}, Matthew J Black, {{Haxxx}}  & {{James}}.

```

Reply:
updated this when i rebased